packs/vsphere/actions/vm_hw_power.yaml

```diff
@@ -0,0 +1,23 @@
+---
+  name: "vm_hw_power"
```

Comment:
In this case I prefer 2 separate action. vm_power_on and vm_power_off. Its more obvious from a usage standpoint.